package-overrides/github/angular/bower-material@0.9.0-rc1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "main": "index",
+  "format":"cjs",
+  "registry": "jspm",
+  "dependencies": {
+    "angular": "github:angular/bower-angular@^1.3.15",
+    "angular-animate": "github:angular/bower-angular-animate@^1.3.15",
+    "angular-aria": "github:angular/bower-angular-aria@^1.3.15",
+    "css": "github:systemjs/plugin-css@^0.1.9"
+  },
+  "shim": {
+    "angular-material": {
+      "deps": [ "./angular-material.css!" ]
+    }
+  }
+}
```

Comment:
@guybedford - so what does this shim addition mean exactly? I realize that an Angular Material application requires the `angular-material.css` to be loaded. And we can use the JSPM CSS plugin. But this shim does not load the CSS... so what is the shim doing?

```json
"shim": {
    "angular-material": {
      "deps": [ "./angular-material.css!" ]
    }
  }
```